scss/_growl.scss

```diff
@@ -35,18 +35,26 @@ $component-name: growl;
   }
 
   &.#{$component-name}-default {
-    background: $info;
+    color: $gray-700;
+    background: map-get($alerts-background-colors, success);
+    border: 1px solid map-get($alerts-second-colors, success);
```

Comment:
```suggestion
&.#{$component-name}{
  color: $gray-700;
  &-default {
    background: map-get($alerts-background-colors, success);
    border: 1px solid map-get($alerts-second-colors, success);
  }
```

And applied below

Reply:
Nop, can't do this as the default jquery growl has a color property on every single classes, overwise I would need to use an !important which I don't want to

Reply:
oh ok.